apps/ledger-live-desktop/src/renderer/components/Select/index.tsx

```diff
@@ -115,8 +115,7 @@ class MenuList<
     const initialOffset = options.indexOf(value) * rowHeight;
     const minHeight = Math.min(...[maxHeight, rowHeight * children.length]);
     if (!children.length && noOptionsMessage) {
-      // @ts-expect-error ts says innerProps is missing but I'm not sure adding it is a good idea
-      return <components.NoOptionsMessage {...this.props} />;
+      return <components.NoOptionsMessage {...this.props} innerProps={{}} />;
```

Comment:
This `innerProps` resolves itself as a spread on a div. It is safe to add. https://github.com/JedWatson/react-select/blob/react-select%403.2.0/packages/react-select/src/components/Menu.js#L436C22-L436C22